### PR TITLE
Allow setting the pooling time in DLPHook

### PR DESCRIPTION
**Pull request by xuan616**
Issue link: #8822 
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by mik-laj** [comment]
What value do you use most often?

**Comment by crazy-2020** [comment]
> What value do you use most often?

It depends on how long it takes for DLP jobs to complete. The more infotype we use, the longer time it takes. The bigger size you scan, the long time it takes. Like a 10.62 MiB, with 15 infotype can took about 3 minutes. A 100 MB would probably took 10m-20min. A simplest job would still take more than 1m. So I think 60s is a reasonable default val and a configurable 'time_to_sleep_in_seconds' is necessary. Thanks.

**Comment by crazy-2020** [comment]
Hi @mik-laj , Could you please help review this PR? Thanks.

**Comment by mik-laj** [comment]
Hej. Can you change default values to 60s? Cheers

File: airflow/providers/google/cloud/hooks/dlp.py [hunk 1]
```diff
@@ -180,6 +178,7 @@ def create_dlp_job(
         timeout: Optional[float] = None,
         metadata: Optional[Sequence[Tuple[str, str]]] = None,
         wait_until_finished: bool = True,
+        time_to_sleep_in_seconds: int = 1
```
Comment (mik-laj) [inline airflow/providers/google/cloud/hooks/dlp.py]:
```suggestion
        time_to_sleep_in_seconds: int = 60
```

File: airflow/providers/google/cloud/hooks/dlp.py [hunk 2]
```diff
@@ -207,6 +206,9 @@ def create_dlp_job(
             until it is set to DONE.
         :type wait_until_finished: bool
         :rtype: google.cloud.dlp_v2.types.DlpJob
+        :param time_to_sleep_in_seconds: (Optional) Time to sleep between active checks
+            of the operation results. Defaults to 1.
```
Comment (mik-laj) [inline airflow/providers/google/cloud/hooks/dlp.py]:
```suggestion
            of the operation results. Defaults to 60s.
```

Comment (ashb) [inline airflow/providers/google/cloud/hooks/dlp.py]:
```suggestion
        :param time_to_sleep_in_seconds: (Optional) Time to sleep, in seconds, between active checks
            of the operation results. Defaults to 60.
```

Comment (ashb) [reply airflow/providers/google/cloud/hooks/dlp.py]: (yes, it's in the name already).

If we say "default to 60s" then someone may try to put `"5m" `, or another string value there if they aren't paying close attention

Comment (xuan616) [reply airflow/providers/google/cloud/hooks/dlp.py]: Thanks for the suggested change.

File: airflow/providers/google/cloud/hooks/dlp.py (hunk 1 above)
Comment (xuan616) [reply airflow/providers/google/cloud/hooks/dlp.py]: Thanks, 60 helps a lot

File: airflow/providers/google/cloud/hooks/dlp.py (hunk 2 above)
Comment (xuan616) [reply airflow/providers/google/cloud/hooks/dlp.py]: Thanks, the new docstring will dispel the potential confusion.

**Review by mik-laj — APPROVED** [review]

**Review by mik-laj — APPROVED** [review]

**Pull request merged**